test/functional/Java21andUp/src/org/openj9/test/jep442/downcall/PrimitiveTypeTests2.java

```diff
@@ -301,4 +301,13 @@ public void test_printfFromDefaultLibWithMemAddr_LinkerOption_2() throws Throwab
 		MemorySegment formatSegmt = arena.allocateUtf8String("\n%d + %d = %d\n");
 		mh.invoke(functionSymbol, formatSegmt, 15, 27, 42);
 	}
+
+	@Test
+	public void test_validateTrivialOption_2() throws Throwable {
```

Comment:
This seems a duplicate of `PrimitiveTypeTests1.test_validateTrivialOption_1()`; why do we need both?

Reply:
Not really. The downcall address (`functionSymbol`) is passed differently between `_1` (passed in the preparation of data) 
```
MethodHandle mh = linker.downcallHandle(functionSymbol, <-------
                                                                        fd, Linker.Option.isTrivial());
int result = (int)mh.invokeExact(111);
```
and `_2` (passed in the the invocation of MH)
```
MethodHandle mh = linker.downcallHandle(fd, Linker.Option.isTrivial());
int result = (int)mh.invokeExact(functionSymbol, <--------
                                                    111);
```
So the newly added tests are to ensue the new code works good in both cases without any issue.